Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/audit/AuditDataTransformHelper.java

```diff
@@ -143,29 +146,38 @@ public static CodedValueType createCodeValueType(String code, String codeSys, St
         CodedValueType codeValueType = new CodedValueType();
 
         // Set the Code
-        codeValueType.setCode(code);
+        if (NullChecker.isNotNullish(code)) {
```

Comment:
What is the value of null-checking these values?  Do we meaningfully distinguish between the default value, blank, and null anywhere in the code?